rcsbapi/data/query.py

```diff
@@ -5,15 +5,32 @@
 from typing import Any, Union, List, Dict, Optional
 import requests
 from rcsbapi.data import SCHEMA
+from .constants import ApiSettings, SINGULAR_TO_PLURAL, ID_TO_SEPARATOR
 
-PDB_URL = "https://data.rcsb.org/graphql"
 logger = logging.getLogger()
 logging.basicConfig(level=logging.INFO, format="[%(levelname)s]: %(message)s")
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
should be done in tandem with other logging related suggestions (see below)